_re2.cc

```diff
@@ -372,7 +390,7 @@ create_regexp(PyObject* pattern)
 
   const std::map<std::string, int>& name_map = regexp->re2_obj->NamedCapturingGroups();
   for (std::map<std::string, int>::const_iterator it = name_map.begin(); it != name_map.end(); ++it) {
-    PyObject* index = PyInt_FromLong(it->second);
+    PyObject* index = PyLong_FromLong(it->second);
```

Comment:
The return value will be a long() instead of an int().  It's possible that we don't want that API to change in 2, so it may be better to define as a macro conditionally based on `#if PY_MAJOR_VERS`.

Reply:
Still needs a comment here, I think.